tinygrad/helpers.py

```diff
@@ -3,6 +3,8 @@
 
 def dedup(x): return list(dict.fromkeys(x))   # retains list order
 def prod(x): return math.prod(x)
+def idxfix(i, sz): return sz+i if i < 0 else i
```

Comment:
I don't think these are generic enough to go in helpers, will they be used anywhere else?

Reply:
just slcfix now, I can move it to the bottom of tensor.py if you like?

Reply:
Why would you put it at the bottom? Keep it local!

Reply:
fixed